File: April/20th Apr 2022/q2.java
// 2. Factorial

// { Driver Code Starts
//Initial Template for Java

import java.util.Scanner;

class q2 {
    
// } Driver Code Ends
//User function Template for Java

public static int nFactorial(int n){
    int ans = 1;
    
    for(int i = 1; i <= n; i++)
        ans = ans * i;
    
    return ans;
}

// { Driver Code Starts.

    public static void main(String[] args)
    {
        Scanner scn = new Scanner(System.in);
        int t = scn.nextInt();
        while(t-- > 0) {
            int n = scn.nextInt();
            int ans = nFactorial(n);
            System.out.println(ans);
        }
        scn.close();
    }
}  // } Driver Code Ends

File: April/17th Apr 2022/q2.cpp
// 2. Distinct Sorted

// { Driver Code Starts
// Initial Template for C++
#include <bits/stdc++.h>
using namespace std;

// } Driver Code Ends
// User function Template for C++

class Solution
{
public:
    // Complete this function
    vector<int> uniqueSorted(int arr[], int n)
    {
        set<int> s;
        for (int i = 0; i < n; i++)
        {
            s.insert(arr[i]);
        }
        vector<int> v(s.begin(), s.end()); // copying set elements into vector
        return v;
    }
};

// { Driver Code Starts.
int main()
{
    int t;
    cin >> t;
    while (t--)
    {
        int n;
        cin >> n;
        int arr[n];

        for (int i = 0; i < n; i++)
            cin >> arr[i];
        Solution ob;
        vector<int> ans = ob.uniqueSorted(arr, n);
        for (auto i : ans)
            cout << i << " ";
        cout << endl;
    }
    return 0;
}
// } Driver Code Ends